util/browser.js

```diff
@@ -143,11 +143,13 @@ export function chromeArgs (proxy, userAgent=null, extraArgs=[]) {
 }
 
 
-export async function evaluateWithCLI(frame, funcString, name = "behaviors") {
+export async function evaluateWithCLI(frame, funcString, logData, contextName) {
   const context = await frame.executionContext();
   const url = frame.url();
 
-  logger.info(`Running ${name}...`, {url});
+  let details = {url, ...logData};
```

Comment:
This results in lines like:

```
{"logLevel":"info","timestamp":"2023-02-23T22:14:17.355Z","context":"behavior","message":"Run Script Finished","details":{"url":"https://www.bitarchivist.net/","page":"https://bitarchivist.net/"}}
```

We can probably remove the `url` field, since it duplicates `page`?

Reply:
Yeah, for top level frame, it will be the same, but for iframes, the url would be of the iframe.
Perhaps only add if it is different?

Reply:
And to be clearer, can call it `frameUrl`.

Reply:
with the name change I think it's fine to have both